src/HttpKernel/View/View.php

```diff
@@ -18,31 +18,27 @@ final class View
      */
     public $result;
 
-    /**
-     * @var array
-     */
-    public $headers;
+    public array $headers;
 
-    /**
-     * @var int
-     */
-    public $statusCode;
+    public int $statusCode;
 
     /**
      * @var string[]|null
      */
-    public $serializationGroups;
+    public ?array $serializationGroups;
 
     /**
      * @var Type|null
```

Comment:
I left it because saying just because it is a string array and not an array that can contain any value (obviously PHP permits any element, but it is clearer for the reader)

Reply:
The comment was on `Type|null` declaration

Reply:
ah, ok :D